app/services/irs_attempts_api/tracker.rb

```diff
@@ -82,6 +82,10 @@ def enabled?
       IdentityConfig.store.irs_attempt_api_enabled && @enabled_for_session
     end
 
+    def ignore_idv_event?(event_type)
+      !IdentityConfig.store.irs_attempt_api_idv_events_enabled && (event_type.to_s.include? 'idv_')
+    end
```

Comment:
1. I'm not a huge fan of the implicit check `idv_` but this is less error-prone than adding manually to every single event and hoping that future us names them correctly
2. Small suggestion to use `start_with?`
```suggestion
    def ignore_idv_event?(event_type)
      !IdentityConfig.store.irs_attempt_api_idv_events_enabled && event_type.to_s.start_with?('idv_')
    end
```